apps/mobile/utils/bridgeHandler.ts

```diff
@@ -0,0 +1,53 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { WEBVIEW_MESSAGE_TYPES } from "@repo/constants";
+import { RefObject } from "react";
+import { Platform } from "react-native";
+import WebView from "react-native-webview";
+import { WebViewProgressEvent, WebViewMessageEvent } from "react-native-webview/lib/WebViewTypes";
+
+import webviewLoginBridge from "@/utils/webviewLoginBridge";
+
+import { parseMessage } from "./parseMessage";
+import { sendMessageToWeb } from "./sendMessageToWeb";
+
+const isIos = Platform.OS === "ios";
+const isAndroid = Platform.OS === "android";
+
+const sendData = async (webViewRef: RefObject<WebView<object>>) => {
```

Comment:
```suggestion
const sendAuthData = async (webViewRef: RefObject<WebView<object>>) => {
```
범용적으로 사용되는건 아닌거같아서 네이밍이 조금 더 명확해도 좋을거같은데